README.md

```diff
@@ -17,6 +17,17 @@
 In your Nextcloud, simply navigate to »Apps«, choose the category »Organization«, find the Tasks app and enable it.
 Then open the Tasks app from the app menu.
 
+## Third-party clients
```

Comment:
```suggestion
## Apps which sync with Nextcloud Tasks (using CalDAV)
```

Reply:
Fine with me

Reply:
@sunjam Could you please apply this change (I don't have permission to push to your repo), then we can merge.

Reply:
@raimund-schluessler you can always push to a pr on github (unless the user explicitly disabled it)
You can use https://github.com/github/hub and checkout pr directly :)
`hub pr checkout 650`

Reply:
Interesting, I just tried to commit the suggestion using the button above, which didn't work.

Reply:
Ah yes, you're right, it seems that this web feature here does not work indeed :)